scripts/llm/ptq.py

```diff
@@ -85,19 +86,25 @@ def get_args():
     parser.add_argument(
         '--generate_sample', help='Generate sample model output after performing PTQ', action='store_true'
     )
+    parser.set_defaults(generate_sample=False)
```

Comment:
There is no need for `parser.set_defaults(generate_sample=False)` and `parser.set_defaults(trust_remote_code=False)`, they are `False` by default.